## 02_u-deva_common_data.diff by Brian Nelson <pyro@debian.org>
##
## 
--- a/Makefile.am
+++ b/Makefile.am
@@ -385,7 +385,8 @@
   data/iso-8859-16.cmap data/iso-8859-16.cset \
   data/koi8-r.cmap data/koi8-r.cset \
   data/koi8-u.cmap data/koi8-u.cset \
-  data/dvorak.kbd data/split.kbd data/standard.kbd
+  data/dvorak.kbd data/split.kbd data/standard.kbd \
+  data/u-deva.cmap data/u-deva.cset
 
 EXTRA_DIST += config.rpath  README ${pkgdata_DATA} m4/*.m4 \
               misc/po-filter.c
--- /dev/null
+++ b/data/u-deva.cmap
@@ -0,0 +1,1362 @@
+INTERNAL
+/
+N 334
+0000 > 00
+0001 > 01
+0002 > 02
+0003 > 03
+0004 > 04
+0005 > 05
+0006 > 06
+0007 > 07
+0008 > 08
+0009 > 09
+000A > 0A
+000B > 0B
+000C > 0C
+000D > 0D
+000E > 0E
+000F > 0F
+0010 > 10
+0011 > 11
+0012 > 12
+0013 > 13
+0014 > 14
+0015 > 15
+0016 > 16
+0017 > 17
+0018 > 18
+0019 > 19
+001A > 1A
+001B > 1B
+001C > 1C
+001D > 1D
+001E > 1E
+001F > 1F
+0020 > 20
+0021 > 21
+0022 > 22
+0023 > 23
+0024 > 24
+0025 > 25
+0026 > 26
+0027 > 27
+0028 > 28
+0029 > 29
+002A > 2A
+002B > 2B
+002C > 2C
+002D > 2D
+002E > 2E
+002F > 2F
+0030 > 30
+0031 > 31
+0032 > 32
+0033 > 33
+0034 > 34
+0035 > 35
+0036 > 36
+0037 > 37
+0038 > 38
+0039 > 39
+003A > 3A
+003B > 3B
+003C > 3C
+003D > 3D
+003E > 3E
+003F > 3F
+0040 > 40
+0041 > 41
+0042 > 42
+0043 > 43
+0044 > 44
+0045 > 45
+0046 > 46
+0047 > 47
+0048 > 48
+0049 > 49
+004A > 4A
+004B > 4B
+004C > 4C
+004D > 4D
+004E > 4E
+004F > 4F
+0050 > 50
+0051 > 51
+0052 > 52
+0053 > 53
+0054 > 54
+0055 > 55
+0056 > 56
+0057 > 57
+0058 > 58
+0059 > 59
+005A > 5A
+005B > 5B
+005C > 5C
+005D > 5D
+005E > 5E
+005F > 5F
+0060 > 60
+0061 > 61
+0062 > 62
+0063 > 63
+0064 > 64
+0065 > 65
+0066 > 66
+0067 > 67
+0068 > 68
+0069 > 69
+006A > 6A
+006B > 6B
+006C > 6C
+006D > 6D
+006E > 6E
+006F > 6F
+0070 > 70
+0071 > 71
+0072 > 72
+0073 > 73
+0074 > 74
+0075 > 75
+0076 > 76
+0077 > 77
+0078 > 78
+0079 > 79
+007A > 7A
+007B > 7B
+007C > 7C
+007D > 7D
+007E > 7E
+007F > 7F
+0080 > 80
+00A0 > 20 # compat
+037E > 3B
+0901 > 81
+0902 > 82
+0903 > 83
+0904 > 84
+0905 > 85
+0906 > 86
+0907 > 87
+0908 > 88
+0909 > 89
+090A > 8A
+090B > 8B
+090C > 8C
+090D > 8D
+090E > 8E
+090F > 8F
+0910 > 90
+0911 > 91
+0912 > 92
+0913 > 93
+0914 > 94
+0915 > 95 /
+     N 1
+     093C > D8
+     .
+0916 > 96 /
+     N 1
+     093C > D9
+     .
+0917 > 97 /
+     N 1
+     093C > DA
+     .
+0918 > 98
+0919 > 99
+091A > 9A
+091B > 9B
+091C > 9C /
+     N 1
+     093C > DB
+     .
+091D > 9D
+091E > 9E
+091F > 9F
+0920 > A0
+0921 > A1 /
+     N 1
+     093C > DC
+     .
+0922 > A2 /
+     N 1
+     093C > DD
+     .
+0923 > A3
+0924 > A4
+0925 > A5
+0926 > A6
+0927 > A7
+0928 > A8 /
+     N 1
+     093C > A9
+     .
+0929 > A9
+092A > AA
+092B > AB /
+     N 1
+     093C > DE
+     .
+092C > AC
+092D > AD
+092E > AE
+092F > AF /
+     N 1
+     093C > DF
+     .
+0930 > B0 /
+     N 1
+     093C > B1
+     .
+0931 > B1
+0932 > B2
+0933 > B3 /
+     N 1
+     093C > B4
+     .
+0934 > B4
+0935 > B5
+0936 > B6
+0937 > B7
+0938 > B8
+0939 > B9
+093C > BC
+093D > BD
+093E > BE
+093F > BF
+0940 > C0
+0941 > C1
+0942 > C2
+0943 > C3
+0944 > C4
+0945 > C5
+0946 > C6
+0947 > C7
+0948 > C8
+0949 > C9
+094A > CA
+094B > CB
+094C > CC
+094D > CD
+0950 > D0
+0951 > D1
+0952 > D2
+0953 > D3
+0954 > D4
+0958 > D8
+0959 > D9
+095A > DA
+095B > DB
+095C > DC
+095D > DD
+095E > DE
+095F > DF
+0960 > E0
+0961 > E1
+0962 > E2
+0963 > E3
+0964 > E4
+0965 > E5
+0966 > E6
+0967 > E7
+0968 > E8
+0969 > E9
+096A > EA
+096B > EB
+096C > EC
+096D > ED
+096E > EE
+096F > EF
+0970 > F0
+1FEF > 60
+2000 > 20 # compat
+2001 > 20 # compat
+2002 > 20 # compat
+2003 > 20 # compat
+2004 > 20 # compat
+2005 > 20 # compat
+2006 > 20 # compat
+2007 > 20 # compat
+2008 > 20 # compat
+2009 > 20 # compat
+200A > 20 # compat
+2024 > 2E # compat
+2025 > 2E 2E # compat
+2026 > 2E 2E 2E # compat
+202F > 20 # compat
+203C > 21 21 # compat
+2047 > 3F 3F # compat
+2048 > 3F 21 # compat
+2049 > 21 3F # compat
+205F > 20 # compat
+FB29 > 2B # compat
+FE30 > 2E 2E # compat
+FE33 > 5F # compat
+FE34 > 5F # compat
+FE35 > 28 # compat
+FE36 > 29 # compat
+FE37 > 7B # compat
+FE38 > 7D # compat
+FE47 > 5B # compat
+FE48 > 5D # compat
+FE4D > 5F # compat
+FE4E > 5F # compat
+FE4F > 5F # compat
+FE50 > 2C # compat
+FE52 > 2E # compat
+FE54 > 3B # compat
+FE55 > 3A # compat
+FE56 > 3F # compat
+FE57 > 21 # compat
+FE59 > 28 # compat
+FE5A > 29 # compat
+FE5B > 7B # compat
+FE5C > 7D # compat
+FE5F > 23 # compat
+FE60 > 26 # compat
+FE61 > 2A # compat
+FE62 > 2B # compat
+FE63 > 2D # compat
+FE64 > 3C # compat
+FE65 > 3E # compat
+FE66 > 3D # compat
+FE68 > 5C # compat
+FE69 > 24 # compat
+FE6A > 25 # compat
+FE6B > 40 # compat
+FF01 > 21 # compat
+FF02 > 22 # compat
+FF03 > 23 # compat
+FF04 > 24 # compat
+FF05 > 25 # compat
+FF06 > 26 # compat
+FF07 > 27 # compat
+FF08 > 28 # compat
+FF09 > 29 # compat
+FF0A > 2A # compat
+FF0B > 2B # compat
+FF0C > 2C # compat
+FF0D > 2D # compat
+FF0E > 2E # compat
+FF0F > 2F # compat
+FF10 > 30 # compat
+FF11 > 31 # compat
+FF12 > 32 # compat
+FF13 > 33 # compat
+FF14 > 34 # compat
+FF15 > 35 # compat
+FF16 > 36 # compat
+FF17 > 37 # compat
+FF18 > 38 # compat
+FF19 > 39 # compat
+FF1A > 3A # compat
+FF1B > 3B # compat
+FF1C > 3C # compat
+FF1D > 3D # compat
+FF1E > 3E # compat
+FF1F > 3F # compat
+FF20 > 40 # compat
+FF3B > 5B # compat
+FF3C > 5C # compat
+FF3D > 5D # compat
+FF3E > 5E # compat
+FF3F > 5F # compat
+FF40 > 60 # compat
+FF5B > 7B # compat
+FF5C > 7C # compat
+FF5D > 7D # compat
+FF5E > 7E # compat
+.
+
+STRICT
+/
+N 236
+0000 > 00
+0001 > 01
+0002 > 02
+0003 > 03
+0004 > 04
+0005 > 05
+0006 > 06
+0007 > 07
+0008 > 08
+0009 > 09
+000A > 0A
+000B > 0B
+000C > 0C
+000D > 0D
+000E > 0E
+000F > 0F
+0010 > 10
+0011 > 11
+0012 > 12
+0013 > 13
+0014 > 14
+0015 > 15
+0016 > 16
+0017 > 17
+0018 > 18
+0019 > 19
+001A > 1A
+001B > 1B
+001C > 1C
+001D > 1D
+001E > 1E
+001F > 1F
+0020 > 20
+0021 > 21
+0022 > 22
+0023 > 23
+0024 > 24
+0025 > 25
+0026 > 26
+0027 > 27
+0028 > 28
+0029 > 29
+002A > 2A
+002B > 2B
+002C > 2C
+002D > 2D
+002E > 2E
+002F > 2F
+0030 > 30
+0031 > 31
+0032 > 32
+0033 > 33
+0034 > 34
+0035 > 35
+0036 > 36
+0037 > 37
+0038 > 38
+0039 > 39
+003A > 3A
+003B > 3B
+003C > 3C
+003D > 3D
+003E > 3E
+003F > 3F
+0040 > 40
+0041 > 41
+0042 > 42
+0043 > 43
+0044 > 44
+0045 > 45
+0046 > 46
+0047 > 47
+0048 > 48
+0049 > 49
+004A > 4A
+004B > 4B
+004C > 4C
+004D > 4D
+004E > 4E
+004F > 4F
+0050 > 50
+0051 > 51
+0052 > 52
+0053 > 53
+0054 > 54
+0055 > 55
+0056 > 56
+0057 > 57
+0058 > 58
+0059 > 59
+005A > 5A
+005B > 5B
+005C > 5C
+005D > 5D
+005E > 5E
+005F > 5F
+0060 > 60
+0061 > 61
+0062 > 62
+0063 > 63
+0064 > 64
+0065 > 65
+0066 > 66
+0067 > 67
+0068 > 68
+0069 > 69
+006A > 6A
+006B > 6B
+006C > 6C
+006D > 6D
+006E > 6E
+006F > 6F
+0070 > 70
+0071 > 71
+0072 > 72
+0073 > 73
+0074 > 74
+0075 > 75
+0076 > 76
+0077 > 77
+0078 > 78
+0079 > 79
+007A > 7A
+007B > 7B
+007C > 7C
+007D > 7D
+007E > 7E
+007F > 7F
+0080 > 80
+037E > 3B
+0901 > 81
+0902 > 82
+0903 > 83
+0904 > 84
+0905 > 85
+0906 > 86
+0907 > 87
+0908 > 88
+0909 > 89
+090A > 8A
+090B > 8B
+090C > 8C
+090D > 8D
+090E > 8E
+090F > 8F
+0910 > 90
+0911 > 91
+0912 > 92
+0913 > 93
+0914 > 94
+0915 > 95 /
+     N 1
+     093C > D8
+     .
+0916 > 96 /
+     N 1
+     093C > D9
+     .
+0917 > 97 /
+     N 1
+     093C > DA
+     .
+0918 > 98
+0919 > 99
+091A > 9A
+091B > 9B
+091C > 9C /
+     N 1
+     093C > DB
+     .
+091D > 9D
+091E > 9E
+091F > 9F
+0920 > A0
+0921 > A1 /
+     N 1
+     093C > DC
+     .
+0922 > A2 /
+     N 1
+     093C > DD
+     .
+0923 > A3
+0924 > A4
+0925 > A5
+0926 > A6
+0927 > A7
+0928 > A8 /
+     N 1
+     093C > A9
+     .
+0929 > A9
+092A > AA
+092B > AB /
+     N 1
+     093C > DE
+     .
+092C > AC
+092D > AD
+092E > AE
+092F > AF /
+     N 1
+     093C > DF
+     .
+0930 > B0 /
+     N 1
+     093C > B1
+     .
+0931 > B1
+0932 > B2
+0933 > B3 /
+     N 1
+     093C > B4
+     .
+0934 > B4
+0935 > B5
+0936 > B6
+0937 > B7
+0938 > B8
+0939 > B9
+093C > BC
+093D > BD
+093E > BE
+093F > BF
+0940 > C0
+0941 > C1
+0942 > C2
+0943 > C3
+0944 > C4
+0945 > C5
+0946 > C6
+0947 > C7
+0948 > C8
+0949 > C9
+094A > CA
+094B > CB
+094C > CC
+094D > CD
+0950 > D0
+0951 > D1
+0952 > D2
+0953 > D3
+0954 > D4
+0958 > D8
+0959 > D9
+095A > DA
+095B > DB
+095C > DC
+095D > DD
+095E > DE
+095F > DF
+0960 > E0
+0961 > E1
+0962 > E2
+0963 > E3
+0964 > E4
+0965 > E5
+0966 > E6
+0967 > E7
+0968 > E8
+0969 > E9
+096A > EA
+096B > EB
+096C > EC
+096D > ED
+096E > EE
+096F > EF
+0970 > F0
+1FEF > 60
+.
+
+NFD
+/
+N 234
+00 > 0000
+01 > 0001
+02 > 0002
+03 > 0003
+04 > 0004
+05 > 0005
+06 > 0006
+07 > 0007
+08 > 0008
+09 > 0009
+0A > 000A
+0B > 000B
+0C > 000C
+0D > 000D
+0E > 000E
+0F > 000F
+10 > 0010
+11 > 0011
+12 > 0012
+13 > 0013
+14 > 0014
+15 > 0015
+16 > 0016
+17 > 0017
+18 > 0018
+19 > 0019
+1A > 001A
+1B > 001B
+1C > 001C
+1D > 001D
+1E > 001E
+1F > 001F
+20 > 0020
+21 > 0021
+22 > 0022
+23 > 0023
+24 > 0024
+25 > 0025
+26 > 0026
+27 > 0027
+28 > 0028
+29 > 0029
+2A > 002A
+2B > 002B
+2C > 002C
+2D > 002D
+2E > 002E
+2F > 002F
+30 > 0030
+31 > 0031
+32 > 0032
+33 > 0033
+34 > 0034
+35 > 0035
+36 > 0036
+37 > 0037
+38 > 0038
+39 > 0039
+3A > 003A
+3B > 003B
+3C > 003C
+3D > 003D
+3E > 003E
+3F > 003F
+40 > 0040
+41 > 0041
+42 > 0042
+43 > 0043
+44 > 0044
+45 > 0045
+46 > 0046
+47 > 0047
+48 > 0048
+49 > 0049
+4A > 004A
+4B > 004B
+4C > 004C
+4D > 004D
+4E > 004E
+4F > 004F
+50 > 0050
+51 > 0051
+52 > 0052
+53 > 0053
+54 > 0054
+55 > 0055
+56 > 0056
+57 > 0057
+58 > 0058
+59 > 0059
+5A > 005A
+5B > 005B
+5C > 005C
+5D > 005D
+5E > 005E
+5F > 005F
+60 > 0060
+61 > 0061
+62 > 0062
+63 > 0063
+64 > 0064
+65 > 0065
+66 > 0066
+67 > 0067
+68 > 0068
+69 > 0069
+6A > 006A
+6B > 006B
+6C > 006C
+6D > 006D
+6E > 006E
+6F > 006F
+70 > 0070
+71 > 0071
+72 > 0072
+73 > 0073
+74 > 0074
+75 > 0075
+76 > 0076
+77 > 0077
+78 > 0078
+79 > 0079
+7A > 007A
+7B > 007B
+7C > 007C
+7D > 007D
+7E > 007E
+7F > 007F
+80 > 0080
+81 > 0901
+82 > 0902
+83 > 0903
+84 > 0904
+85 > 0905
+86 > 0906
+87 > 0907
+88 > 0908
+89 > 0909
+8A > 090A
+8B > 090B
+8C > 090C
+8D > 090D
+8E > 090E
+8F > 090F
+90 > 0910
+91 > 0911
+92 > 0912
+93 > 0913
+94 > 0914
+95 > 0915
+96 > 0916
+97 > 0917
+98 > 0918
+99 > 0919
+9A > 091A
+9B > 091B
+9C > 091C
+9D > 091D
+9E > 091E
+9F > 091F
+A0 > 0920
+A1 > 0921
+A2 > 0922
+A3 > 0923
+A4 > 0924
+A5 > 0925
+A6 > 0926
+A7 > 0927
+A8 > 0928
+A9 > 0928 093C
+AA > 092A
+AB > 092B
+AC > 092C
+AD > 092D
+AE > 092E
+AF > 092F
+B0 > 0930
+B1 > 0930 093C
+B2 > 0932
+B3 > 0933
+B4 > 0933 093C
+B5 > 0935
+B6 > 0936
+B7 > 0937
+B8 > 0938
+B9 > 0939
+BC > 093C
+BD > 093D
+BE > 093E
+BF > 093F
+C0 > 0940
+C1 > 0941
+C2 > 0942
+C3 > 0943
+C4 > 0944
+C5 > 0945
+C6 > 0946
+C7 > 0947
+C8 > 0948
+C9 > 0949
+CA > 094A
+CB > 094B
+CC > 094C
+CD > 094D
+D0 > 0950
+D1 > 0951
+D2 > 0952
+D3 > 0953
+D4 > 0954
+D8 > 0915 093C
+D9 > 0916 093C
+DA > 0917 093C
+DB > 091C 093C
+DC > 0921 093C
+DD > 0922 093C
+DE > 092B 093C
+DF > 092F 093C
+E0 > 0960
+E1 > 0961
+E2 > 0962
+E3 > 0963
+E4 > 0964
+E5 > 0965
+E6 > 0966
+E7 > 0967
+E8 > 0968
+E9 > 0969
+EA > 096A
+EB > 096B
+EC > 096C
+ED > 096D
+EE > 096E
+EF > 096F
+F0 > 0970
+.
+
+NFC
+/
+N 234
+00 > 0000
+01 > 0001
+02 > 0002
+03 > 0003
+04 > 0004
+05 > 0005
+06 > 0006
+07 > 0007
+08 > 0008
+09 > 0009
+0A > 000A
+0B > 000B
+0C > 000C
+0D > 000D
+0E > 000E
+0F > 000F
+10 > 0010
+11 > 0011
+12 > 0012
+13 > 0013
+14 > 0014
+15 > 0015
+16 > 0016
+17 > 0017
+18 > 0018
+19 > 0019
+1A > 001A
+1B > 001B
+1C > 001C
+1D > 001D
+1E > 001E
+1F > 001F
+20 > 0020
+21 > 0021
+22 > 0022
+23 > 0023
+24 > 0024
+25 > 0025
+26 > 0026
+27 > 0027
+28 > 0028
+29 > 0029
+2A > 002A
+2B > 002B
+2C > 002C
+2D > 002D
+2E > 002E
+2F > 002F
+30 > 0030
+31 > 0031
+32 > 0032
+33 > 0033
+34 > 0034
+35 > 0035
+36 > 0036
+37 > 0037
+38 > 0038
+39 > 0039
+3A > 003A
+3B > 003B
+3C > 003C
+3D > 003D
+3E > 003E
+3F > 003F
+40 > 0040
+41 > 0041
+42 > 0042
+43 > 0043
+44 > 0044
+45 > 0045
+46 > 0046
+47 > 0047
+48 > 0048
+49 > 0049
+4A > 004A
+4B > 004B
+4C > 004C
+4D > 004D
+4E > 004E
+4F > 004F
+50 > 0050
+51 > 0051
+52 > 0052
+53 > 0053
+54 > 0054
+55 > 0055
+56 > 0056
+57 > 0057
+58 > 0058
+59 > 0059
+5A > 005A
+5B > 005B
+5C > 005C
+5D > 005D
+5E > 005E
+5F > 005F
+60 > 0060
+61 > 0061
+62 > 0062
+63 > 0063
+64 > 0064
+65 > 0065
+66 > 0066
+67 > 0067
+68 > 0068
+69 > 0069
+6A > 006A
+6B > 006B
+6C > 006C
+6D > 006D
+6E > 006E
+6F > 006F
+70 > 0070
+71 > 0071
+72 > 0072
+73 > 0073
+74 > 0074
+75 > 0075
+76 > 0076
+77 > 0077
+78 > 0078
+79 > 0079
+7A > 007A
+7B > 007B
+7C > 007C
+7D > 007D
+7E > 007E
+7F > 007F
+80 > 0080
+81 > 0901
+82 > 0902
+83 > 0903
+84 > 0904
+85 > 0905
+86 > 0906
+87 > 0907
+88 > 0908
+89 > 0909
+8A > 090A
+8B > 090B
+8C > 090C
+8D > 090D
+8E > 090E
+8F > 090F
+90 > 0910
+91 > 0911
+92 > 0912
+93 > 0913
+94 > 0914
+95 > 0915
+96 > 0916
+97 > 0917
+98 > 0918
+99 > 0919
+9A > 091A
+9B > 091B
+9C > 091C
+9D > 091D
+9E > 091E
+9F > 091F
+A0 > 0920
+A1 > 0921
+A2 > 0922
+A3 > 0923
+A4 > 0924
+A5 > 0925
+A6 > 0926
+A7 > 0927
+A8 > 0928
+A9 > 0929
+AA > 092A
+AB > 092B
+AC > 092C
+AD > 092D
+AE > 092E
+AF > 092F
+B0 > 0930
+B1 > 0931
+B2 > 0932
+B3 > 0933
+B4 > 0934
+B5 > 0935
+B6 > 0936
+B7 > 0937
+B8 > 0938
+B9 > 0939
+BC > 093C
+BD > 093D
+BE > 093E
+BF > 093F
+C0 > 0940
+C1 > 0941
+C2 > 0942
+C3 > 0943
+C4 > 0944
+C5 > 0945
+C6 > 0946
+C7 > 0947
+C8 > 0948
+C9 > 0949
+CA > 094A
+CB > 094B
+CC > 094C
+CD > 094D
+D0 > 0950
+D1 > 0951
+D2 > 0952
+D3 > 0953
+D4 > 0954
+D8 > 0915 093C
+D9 > 0916 093C
+DA > 0917 093C
+DB > 091C 093C
+DC > 0921 093C
+DD > 0922 093C
+DE > 092B 093C
+DF > 092F 093C
+E0 > 0960
+E1 > 0961
+E2 > 0962
+E3 > 0963
+E4 > 0964
+E5 > 0965
+E6 > 0966
+E7 > 0967
+E8 > 0968
+E9 > 0969
+EA > 096A
+EB > 096B
+EC > 096C
+ED > 096D
+EE > 096E
+EF > 096F
+F0 > 0970
+.
+
+COMP
+/
+N 234
+00 > 0000
+01 > 0001
+02 > 0002
+03 > 0003
+04 > 0004
+05 > 0005
+06 > 0006
+07 > 0007
+08 > 0008
+09 > 0009
+0A > 000A
+0B > 000B
+0C > 000C
+0D > 000D
+0E > 000E
+0F > 000F
+10 > 0010
+11 > 0011
+12 > 0012
+13 > 0013
+14 > 0014
+15 > 0015
+16 > 0016
+17 > 0017
+18 > 0018
+19 > 0019
+1A > 001A
+1B > 001B
+1C > 001C
+1D > 001D
+1E > 001E
+1F > 001F
+20 > 0020
+21 > 0021
+22 > 0022
+23 > 0023
+24 > 0024
+25 > 0025
+26 > 0026
+27 > 0027
+28 > 0028
+29 > 0029
+2A > 002A
+2B > 002B
+2C > 002C
+2D > 002D
+2E > 002E
+2F > 002F
+30 > 0030
+31 > 0031
+32 > 0032
+33 > 0033
+34 > 0034
+35 > 0035
+36 > 0036
+37 > 0037
+38 > 0038
+39 > 0039
+3A > 003A
+3B > 003B
+3C > 003C
+3D > 003D
+3E > 003E
+3F > 003F
+40 > 0040
+41 > 0041
+42 > 0042
+43 > 0043
+44 > 0044
+45 > 0045
+46 > 0046
+47 > 0047
+48 > 0048
+49 > 0049
+4A > 004A
+4B > 004B
+4C > 004C
+4D > 004D
+4E > 004E
+4F > 004F
+50 > 0050
+51 > 0051
+52 > 0052
+53 > 0053
+54 > 0054
+55 > 0055
+56 > 0056
+57 > 0057
+58 > 0058
+59 > 0059
+5A > 005A
+5B > 005B
+5C > 005C
+5D > 005D
+5E > 005E
+5F > 005F
+60 > 0060
+61 > 0061
+62 > 0062
+63 > 0063
+64 > 0064
+65 > 0065
+66 > 0066
+67 > 0067
+68 > 0068
+69 > 0069
+6A > 006A
+6B > 006B
+6C > 006C
+6D > 006D
+6E > 006E
+6F > 006F
+70 > 0070
+71 > 0071
+72 > 0072
+73 > 0073
+74 > 0074
+75 > 0075
+76 > 0076
+77 > 0077
+78 > 0078
+79 > 0079
+7A > 007A
+7B > 007B
+7C > 007C
+7D > 007D
+7E > 007E
+7F > 007F
+80 > 0080
+81 > 0901
+82 > 0902
+83 > 0903
+84 > 0904
+85 > 0905
+86 > 0906
+87 > 0907
+88 > 0908
+89 > 0909
+8A > 090A
+8B > 090B
+8C > 090C
+8D > 090D
+8E > 090E
+8F > 090F
+90 > 0910
+91 > 0911
+92 > 0912
+93 > 0913
+94 > 0914
+95 > 0915
+96 > 0916
+97 > 0917
+98 > 0918
+99 > 0919
+9A > 091A
+9B > 091B
+9C > 091C
+9D > 091D
+9E > 091E
+9F > 091F
+A0 > 0920
+A1 > 0921
+A2 > 0922
+A3 > 0923
+A4 > 0924
+A5 > 0925
+A6 > 0926
+A7 > 0927
+A8 > 0928
+A9 > 0929
+AA > 092A
+AB > 092B
+AC > 092C
+AD > 092D
+AE > 092E
+AF > 092F
+B0 > 0930
+B1 > 0931
+B2 > 0932
+B3 > 0933
+B4 > 0934
+B5 > 0935
+B6 > 0936
+B7 > 0937
+B8 > 0938
+B9 > 0939
+BC > 093C
+BD > 093D
+BE > 093E
+BF > 093F
+C0 > 0940
+C1 > 0941
+C2 > 0942
+C3 > 0943
+C4 > 0944
+C5 > 0945
+C6 > 0946
+C7 > 0947
+C8 > 0948
+C9 > 0949
+CA > 094A
+CB > 094B
+CC > 094C
+CD > 094D
+D0 > 0950
+D1 > 0951
+D2 > 0952
+D3 > 0953
+D4 > 0954
+D8 > 0958
+D9 > 0959
+DA > 095A
+DB > 095B
+DC > 095C
+DD > 095D
+DE > 095E
+DF > 095F
+E0 > 0960
+E1 > 0961
+E2 > 0962
+E3 > 0963
+E4 > 0964
+E5 > 0965
+E6 > 0966
+E7 > 0967
+E8 > 0968
+E9 > 0969
+EA > 096A
+EB > 096B
+EC > 096C
+ED > 096D
+EE > 096E
+EF > 096F
+F0 > 0970
+.
--- /dev/null
+++ b/data/u-deva.cset
@@ -0,0 +1,262 @@
+# Aspell Character Data File.
+= u-deva
+> ascii
+/
+# <char> <uni> <type> <display> <upper> <lower> <title> <plain>
+#                                                         <sl-first> <sl-rest>
+00 0000 - N 00 00 00 00 00 00 # <control>
+01 0001 . N 01 01 01 00 00 00 # <unused control>
+02 0002 . N 02 02 02 00 00 00 # <unused control>
+03 0003 . N 03 03 03 00 00 00 # <unused control>
+04 0004 . N 04 04 04 00 00 00 # <unused control>
+05 0005 . N 05 05 05 00 00 00 # <unused control>
+06 0006 . N 06 06 06 00 00 00 # <unused control>
+07 0007 . N 07 07 07 00 00 00 # <unused control>
+08 0008 . N 08 08 08 00 00 00 # <unused control>
+09 0009 . N 09 09 09 00 00 00 # <unused control>
+0A 000A . N 0A 0A 0A 00 00 00 # <unused control>
+0B 000B . N 0B 0B 0B 00 00 00 # <unused control>
+0C 000C . N 0C 0C 0C 00 00 00 # <unused control>
+0D 000D . N 0D 0D 0D 00 00 00 # <unused control>
+0E 000E . N 0E 0E 0E 00 00 00 # <unused control>
+0F 000F . N 0F 0F 0F 00 00 00 # <unused control>
+10 0010 - N 10 10 10 00 00 00 # <control>
+11 0011 . N 11 11 11 00 00 00 # <unused control>
+12 0012 . N 12 12 12 00 00 00 # <unused control>
+13 0013 . N 13 13 13 00 00 00 # <unused control>
+14 0014 . N 14 14 14 00 00 00 # <unused control>
+15 0015 . N 15 15 15 00 00 00 # <unused control>
+16 0016 . N 16 16 16 00 00 00 # <unused control>
+17 0017 . N 17 17 17 00 00 00 # <unused control>
+18 0018 . N 18 18 18 00 00 00 # <unused control>
+19 0019 . N 19 19 19 00 00 00 # <unused control>
+1A 001A . N 1A 1A 1A 00 00 00 # <unused control>
+1B 001B . N 1B 1B 1B 00 00 00 # <unused control>
+1C 001C . N 1C 1C 1C 00 00 00 # <unused control>
+1D 001D . N 1D 1D 1D 00 00 00 # <unused control>
+1E 001E . N 1E 1E 1E 00 00 00 # <unused control>
+1F 001F . N 1F 1F 1F 00 00 00 # <unused control>
+20 0020 W Y 20 20 20 00 00 00 # SPACE
+21 0021 - Y 21 21 21 00 00 00 # EXCLAMATION MARK
+22 0022 - Y 22 22 22 00 00 00 # QUOTATION MARK
+23 0023 - Y 23 23 23 00 00 00 # NUMBER SIGN
+24 0024 - Y 24 24 24 00 00 00 # DOLLAR SIGN
+25 0025 - Y 25 25 25 00 00 00 # PERCENT SIGN
+26 0026 - Y 26 26 26 00 00 00 # AMPERSAND
+27 0027 - Y 27 27 27 00 00 00 # APOSTROPHE
+28 0028 - Y 28 28 28 00 00 00 # LEFT PARENTHESIS
+29 0029 - Y 29 29 29 00 00 00 # RIGHT PARENTHESIS
+2A 002A - Y 2A 2A 2A 00 00 00 # ASTERISK
+2B 002B - Y 2B 2B 2B 00 00 00 # PLUS SIGN
+2C 002C - Y 2C 2C 2C 00 00 00 # COMMA
+2D 002D H Y 2D 2D 2D 00 00 00 # HYPHEN-MINUS
+2E 002E - Y 2E 2E 2E 00 00 00 # FULL STOP
+2F 002F - Y 2F 2F 2F 00 00 00 # SOLIDUS
+30 0030 D Y 30 30 30 00 00 00 # DIGIT ZERO
+31 0031 D Y 31 31 31 00 00 00 # DIGIT ONE
+32 0032 D Y 32 32 32 00 00 00 # DIGIT TWO
+33 0033 D Y 33 33 33 00 00 00 # DIGIT THREE
+34 0034 D Y 34 34 34 00 00 00 # DIGIT FOUR
+35 0035 D Y 35 35 35 00 00 00 # DIGIT FIVE
+36 0036 D Y 36 36 36 00 00 00 # DIGIT SIX
+37 0037 D Y 37 37 37 00 00 00 # DIGIT SEVEN
+38 0038 D Y 38 38 38 00 00 00 # DIGIT EIGHT
+39 0039 D Y 39 39 39 00 00 00 # DIGIT NINE
+3A 003A - Y 3A 3A 3A 00 00 00 # COLON
+3B 003B - Y 3B 3B 3B 00 00 00 # SEMICOLON
+3C 003C - Y 3C 3C 3C 00 00 00 # LESS-THAN SIGN
+3D 003D - Y 3D 3D 3D 00 00 00 # EQUALS SIGN
+3E 003E - Y 3E 3E 3E 00 00 00 # GREATER-THAN SIGN
+3F 003F - Y 3F 3F 3F 00 00 00 # QUESTION MARK
+40 0040 - Y 40 40 40 00 00 00 # COMMERCIAL AT
+41 0041 . Y 41 41 41 00 00 00 # <unused latin letter>
+42 0042 . Y 42 42 42 00 00 00 # <unused latin letter>
+43 0043 . Y 43 43 43 00 00 00 # <unused latin letter>
+44 0044 . Y 44 44 44 00 00 00 # <unused latin letter>
+45 0045 . Y 45 45 45 00 00 00 # <unused latin letter>
+46 0046 . Y 46 46 46 00 00 00 # <unused latin letter>
+47 0047 . Y 47 47 47 00 00 00 # <unused latin letter>
+48 0048 . Y 48 48 48 00 00 00 # <unused latin letter>
+49 0049 . Y 49 49 49 00 00 00 # <unused latin letter>
+4A 004A . Y 4A 4A 4A 00 00 00 # <unused latin letter>
+4B 004B . Y 4B 4B 4B 00 00 00 # <unused latin letter>
+4C 004C . Y 4C 4C 4C 00 00 00 # <unused latin letter>
+4D 004D . Y 4D 4D 4D 00 00 00 # <unused latin letter>
+4E 004E . Y 4E 4E 4E 00 00 00 # <unused latin letter>
+4F 004F . Y 4F 4F 4F 00 00 00 # <unused latin letter>
+50 0050 . Y 50 50 50 00 00 00 # <unused latin letter>
+51 0051 . Y 51 51 51 00 00 00 # <unused latin letter>
+52 0052 . Y 52 52 52 00 00 00 # <unused latin letter>
+53 0053 . Y 53 53 53 00 00 00 # <unused latin letter>
+54 0054 . Y 54 54 54 00 00 00 # <unused latin letter>
+55 0055 . Y 55 55 55 00 00 00 # <unused latin letter>
+56 0056 . Y 56 56 56 00 00 00 # <unused latin letter>
+57 0057 . Y 57 57 57 00 00 00 # <unused latin letter>
+58 0058 . Y 58 58 58 00 00 00 # <unused latin letter>
+59 0059 . Y 59 59 59 00 00 00 # <unused latin letter>
+5A 005A . Y 5A 5A 5A 00 00 00 # <unused latin letter>
+5B 005B - Y 5B 5B 5B 00 00 00 # LEFT SQUARE BRACKET
+5C 005C - Y 5C 5C 5C 00 00 00 # REVERSE SOLIDUS
+5D 005D - Y 5D 5D 5D 00 00 00 # RIGHT SQUARE BRACKET
+5E 005E - Y 5E 5E 5E 00 00 00 # CIRCUMFLEX ACCENT
+5F 005F - Y 5F 5F 5F 00 00 00 # LOW LINE
+60 0060 - Y 60 60 60 00 00 00 # GRAVE ACCENT
+61 0061 . Y 61 61 61 00 00 00 # <unused latin letter>
+62 0062 . Y 62 62 62 00 00 00 # <unused latin letter>
+63 0063 . Y 63 63 63 00 00 00 # <unused latin letter>
+64 0064 . Y 64 64 64 00 00 00 # <unused latin letter>
+65 0065 . Y 65 65 65 00 00 00 # <unused latin letter>
+66 0066 . Y 66 66 66 00 00 00 # <unused latin letter>
+67 0067 . Y 67 67 67 00 00 00 # <unused latin letter>
+68 0068 . Y 68 68 68 00 00 00 # <unused latin letter>
+69 0069 . Y 69 69 69 00 00 00 # <unused latin letter>
+6A 006A . Y 6A 6A 6A 00 00 00 # <unused latin letter>
+6B 006B . Y 6B 6B 6B 00 00 00 # <unused latin letter>
+6C 006C . Y 6C 6C 6C 00 00 00 # <unused latin letter>
+6D 006D . Y 6D 6D 6D 00 00 00 # <unused latin letter>
+6E 006E . Y 6E 6E 6E 00 00 00 # <unused latin letter>
+6F 006F . Y 6F 6F 6F 00 00 00 # <unused latin letter>
+70 0070 . Y 70 70 70 00 00 00 # <unused latin letter>
+71 0071 . Y 71 71 71 00 00 00 # <unused latin letter>
+72 0072 . Y 72 72 72 00 00 00 # <unused latin letter>
+73 0073 . Y 73 73 73 00 00 00 # <unused latin letter>
+74 0074 . Y 74 74 74 00 00 00 # <unused latin letter>
+75 0075 . Y 75 75 75 00 00 00 # <unused latin letter>
+76 0076 . Y 76 76 76 00 00 00 # <unused latin letter>
+77 0077 . Y 77 77 77 00 00 00 # <unused latin letter>
+78 0078 . Y 78 78 78 00 00 00 # <unused latin letter>
+79 0079 . Y 79 79 79 00 00 00 # <unused latin letter>
+7A 007A . Y 7A 7A 7A 00 00 00 # <unused latin letter>
+7B 007B - Y 7B 7B 7B 00 00 00 # LEFT CURLY BRACKET
+7C 007C - Y 7C 7C 7C 00 00 00 # VERTICAL LINE
+7D 007D - Y 7D 7D 7D 00 00 00 # RIGHT CURLY BRACKET
+7E 007E - Y 7E 7E 7E 00 00 00 # TILDE
+7F 007F - N 7F 7F 7F 00 00 00 # <control>
+80 0080 . N 80 80 80 00 00 00 # <unused control>
+81 0901 l M 81 81 81 81 81 81 # DEVANAGARI SIGN CANDRABINDU
+82 0902 l M 82 82 82 82 82 82 # DEVANAGARI SIGN ANUSVARA
+83 0903 l M 83 83 83 83 83 83 # DEVANAGARI SIGN VISARGA
+84 0904 L Y 84 84 84 84 84 84 # DEVANAGARI LETTER SHORT A
+85 0905 L Y 85 85 85 85 85 85 # DEVANAGARI LETTER A
+86 0906 L Y 86 86 86 86 86 86 # DEVANAGARI LETTER AA
+87 0907 L Y 87 87 87 87 87 87 # DEVANAGARI LETTER I
+88 0908 L Y 88 88 88 88 88 88 # DEVANAGARI LETTER II
+89 0909 L Y 89 89 89 89 89 89 # DEVANAGARI LETTER U
+8A 090A L Y 8A 8A 8A 8A 8A 8A # DEVANAGARI LETTER UU
+8B 090B L Y 8B 8B 8B 8B 8B 8B # DEVANAGARI LETTER VOCALIC R
+8C 090C L Y 8C 8C 8C 8C 8C 8C # DEVANAGARI LETTER VOCALIC L
+8D 090D L Y 8D 8D 8D 8D 8D 8D # DEVANAGARI LETTER CANDRA E
+8E 090E L Y 8E 8E 8E 8E 8E 8E # DEVANAGARI LETTER SHORT E
+8F 090F L Y 8F 8F 8F 8F 8F 8F # DEVANAGARI LETTER E
+90 0910 L Y 90 90 90 90 90 90 # DEVANAGARI LETTER AI
+91 0911 L Y 91 91 91 91 91 91 # DEVANAGARI LETTER CANDRA O
+92 0912 L Y 92 92 92 92 92 92 # DEVANAGARI LETTER SHORT O
+93 0913 L Y 93 93 93 93 93 93 # DEVANAGARI LETTER O
+94 0914 L Y 94 94 94 94 94 94 # DEVANAGARI LETTER AU
+95 0915 L Y 95 95 95 95 95 95 # DEVANAGARI LETTER KA
+96 0916 L Y 96 96 96 96 96 96 # DEVANAGARI LETTER KHA
+97 0917 L Y 97 97 97 97 97 97 # DEVANAGARI LETTER GA
+98 0918 L Y 98 98 98 98 98 98 # DEVANAGARI LETTER GHA
+99 0919 L Y 99 99 99 99 99 99 # DEVANAGARI LETTER NGA
+9A 091A L Y 9A 9A 9A 9A 9A 9A # DEVANAGARI LETTER CA
+9B 091B L Y 9B 9B 9B 9B 9B 9B # DEVANAGARI LETTER CHA
+9C 091C L Y 9C 9C 9C 9C 9C 9C # DEVANAGARI LETTER JA
+9D 091D L Y 9D 9D 9D 9D 9D 9D # DEVANAGARI LETTER JHA
+9E 091E L Y 9E 9E 9E 9E 9E 9E # DEVANAGARI LETTER NYA
+9F 091F L Y 9F 9F 9F 9F 9F 9F # DEVANAGARI LETTER TTA
+A0 0920 L Y A0 A0 A0 A0 A0 A0 # DEVANAGARI LETTER TTHA
+A1 0921 L Y A1 A1 A1 A1 A1 A1 # DEVANAGARI LETTER DDA
+A2 0922 L Y A2 A2 A2 A2 A2 A2 # DEVANAGARI LETTER DDHA
+A3 0923 L Y A3 A3 A3 A3 A3 A3 # DEVANAGARI LETTER NNA
+A4 0924 L Y A4 A4 A4 A4 A4 A4 # DEVANAGARI LETTER TA
+A5 0925 L Y A5 A5 A5 A5 A5 A5 # DEVANAGARI LETTER THA
+A6 0926 L Y A6 A6 A6 A6 A6 A6 # DEVANAGARI LETTER DA
+A7 0927 L Y A7 A7 A7 A7 A7 A7 # DEVANAGARI LETTER DHA
+A8 0928 L Y A8 A8 A8 A8 A8 A8 # DEVANAGARI LETTER NA
+A9 0929 L Y A9 A9 A9 A8 A8 A8 # DEVANAGARI LETTER NNNA
+AA 092A L Y AA AA AA AA AA AA # DEVANAGARI LETTER PA
+AB 092B L Y AB AB AB AB AB AB # DEVANAGARI LETTER PHA
+AC 092C L Y AC AC AC AC AC AC # DEVANAGARI LETTER BA
+AD 092D L Y AD AD AD AD AD AD # DEVANAGARI LETTER BHA
+AE 092E L Y AE AE AE AE AE AE # DEVANAGARI LETTER MA
+AF 092F L Y AF AF AF AF AF AF # DEVANAGARI LETTER YA
+B0 0930 L Y B0 B0 B0 B0 B0 B0 # DEVANAGARI LETTER RA
+B1 0931 L Y B1 B1 B1 B0 B0 B0 # DEVANAGARI LETTER RRA
+B2 0932 L Y B2 B2 B2 B2 B2 B2 # DEVANAGARI LETTER LA
+B3 0933 L Y B3 B3 B3 B3 B3 B3 # DEVANAGARI LETTER LLA
+B4 0934 L Y B4 B4 B4 B3 B3 B3 # DEVANAGARI LETTER LLLA
+B5 0935 L Y B5 B5 B5 B5 B5 B5 # DEVANAGARI LETTER VA
+B6 0936 L Y B6 B6 B6 B6 B6 B6 # DEVANAGARI LETTER SHA
+B7 0937 L Y B7 B7 B7 B7 B7 B7 # DEVANAGARI LETTER SSA
+B8 0938 L Y B8 B8 B8 B8 B8 B8 # DEVANAGARI LETTER SA
+B9 0939 L Y B9 B9 B9 B9 B9 B9 # DEVANAGARI LETTER HA
+BA E0BA . N BA BA BA 00 00 00 # <unused>
+BB E0BB . N BB BB BB 00 00 00 # <unused>
+BC 093C M M BC BC BC 00 00 00 # DEVANAGARI SIGN NUKTA
+BD 093D l Y BD BD BD BD BD BD # DEVANAGARI SIGN AVAGRAHA
+BE 093E L M BE BE BE BE 2A 00 # DEVANAGARI VOWEL SIGN AA
+BF 093F L M BF BF BF BF 2A 00 # DEVANAGARI VOWEL SIGN I
+C0 0940 L M C0 C0 C0 C0 2A 00 # DEVANAGARI VOWEL SIGN II
+C1 0941 L M C1 C1 C1 C1 2A 00 # DEVANAGARI VOWEL SIGN U
+C2 0942 L M C2 C2 C2 C2 2A 00 # DEVANAGARI VOWEL SIGN UU
+C3 0943 L M C3 C3 C3 C3 2A 00 # DEVANAGARI VOWEL SIGN VOCALIC R
+C4 0944 L M C4 C4 C4 C4 2A 00 # DEVANAGARI VOWEL SIGN VOCALIC RR
+C5 0945 L M C5 C5 C5 C5 2A 00 # DEVANAGARI VOWEL SIGN CANDRA E
+C6 0946 L M C6 C6 C6 C6 2A 00 # DEVANAGARI VOWEL SIGN SHORT E
+C7 0947 L M C7 C7 C7 C7 2A 00 # DEVANAGARI VOWEL SIGN E
+C8 0948 L M C8 C8 C8 C8 2A 00 # DEVANAGARI VOWEL SIGN AI
+C9 0949 L M C9 C9 C9 C9 2A 00 # DEVANAGARI VOWEL SIGN CANDRA O
+CA 094A L M CA CA CA CA 2A 00 # DEVANAGARI VOWEL SIGN SHORT O
+CB 094B L M CB CB CB CB 2A 00 # DEVANAGARI VOWEL SIGN O
+CC 094C L M CC CC CC CC 2A 00 # DEVANAGARI VOWEL SIGN AU
+CD 094D M M CD CD CD 00 00 00 # DEVANAGARI SIGN VIRAMA
+CE E0CE . N CE CE CE 00 00 00 # <unused>
+CF E0CF . N CF CF CF 00 00 00 # <unused>
+D0 0950 l Y D0 D0 D0 D0 D0 D0 # DEVANAGARI OM
+D1 0951 M M D1 D1 D1 00 00 00 # DEVANAGARI STRESS SIGN UDATTA
+D2 0952 M M D2 D2 D2 00 00 00 # DEVANAGARI STRESS SIGN ANUDATTA
+D3 0953 M M D3 D3 D3 00 00 00 # DEVANAGARI GRAVE ACCENT
+D4 0954 M M D4 D4 D4 00 00 00 # DEVANAGARI ACUTE ACCENT
+D5 E0D5 . N D5 D5 D5 00 00 00 # <unused>
+D6 E0D6 . N D6 D6 D6 00 00 00 # <unused>
+D7 E0D7 . N D7 D7 D7 00 00 00 # <unused>
+D8 0958 L Y D8 D8 D8 95 95 95 # DEVANAGARI LETTER QA
+D9 0959 L Y D9 D9 D9 96 96 96 # DEVANAGARI LETTER KHHA
+DA 095A L Y DA DA DA 97 97 97 # DEVANAGARI LETTER GHHA
+DB 095B L Y DB DB DB 9C 9C 9C # DEVANAGARI LETTER ZA
+DC 095C L Y DC DC DC A1 A1 A1 # DEVANAGARI LETTER DDDHA
+DD 095D L Y DD DD DD A2 A2 A2 # DEVANAGARI LETTER RHA
+DE 095E L Y DE DE DE AB AB AB # DEVANAGARI LETTER FA
+DF 095F L Y DF DF DF AF AF AF # DEVANAGARI LETTER YYA
+E0 0960 L Y E0 E0 E0 E0 E0 E0 # DEVANAGARI LETTER VOCALIC RR
+E1 0961 L Y E1 E1 E1 E1 E1 E1 # DEVANAGARI LETTER VOCALIC LL
+E2 0962 L M E2 E2 E2 E2 2A 00 # DEVANAGARI VOWEL SIGN VOCALIC L
+E3 0963 L M E3 E3 E3 E3 2A 00 # DEVANAGARI VOWEL SIGN VOCALIC LL
+E4 0964 - Y E4 E4 E4 00 00 00 # DEVANAGARI DANDA
+E5 0965 - Y E5 E5 E5 00 00 00 # DEVANAGARI DOUBLE DANDA
+E6 0966 D Y E6 E6 E6 00 00 00 # DEVANAGARI DIGIT ZERO
+E7 0967 D Y E7 E7 E7 00 00 00 # DEVANAGARI DIGIT ONE
+E8 0968 D Y E8 E8 E8 00 00 00 # DEVANAGARI DIGIT TWO
+E9 0969 D Y E9 E9 E9 00 00 00 # DEVANAGARI DIGIT THREE
+EA 096A D Y EA EA EA 00 00 00 # DEVANAGARI DIGIT FOUR
+EB 096B D Y EB EB EB 00 00 00 # DEVANAGARI DIGIT FIVE
+EC 096C D Y EC EC EC 00 00 00 # DEVANAGARI DIGIT SIX
+ED 096D D Y ED ED ED 00 00 00 # DEVANAGARI DIGIT SEVEN
+EE 096E D Y EE EE EE 00 00 00 # DEVANAGARI DIGIT EIGHT
+EF 096F D Y EF EF EF 00 00 00 # DEVANAGARI DIGIT NINE
+F0 0970 - Y F0 F0 F0 00 00 00 # DEVANAGARI ABBREVIATION SIGN
+F1 E0F1 . N F1 F1 F1 00 00 00 # <unused>
+F2 E0F2 . N F2 F2 F2 00 00 00 # <unused>
+F3 E0F3 . N F3 F3 F3 00 00 00 # <unused>
+F4 E0F4 . N F4 F4 F4 00 00 00 # <unused>
+F5 E0F5 . N F5 F5 F5 00 00 00 # <unused>
+F6 E0F6 . N F6 F6 F6 00 00 00 # <unused>
+F7 E0F7 . N F7 F7 F7 00 00 00 # <unused>
+F8 E0F8 . N F8 F8 F8 00 00 00 # <unused>
+F9 E0F9 . N F9 F9 F9 00 00 00 # <unused>
+FA E0FA . N FA FA FA 00 00 00 # <unused>
+FB E0FB . N FB FB FB 00 00 00 # <unused>
+FC E0FC . N FC FC FC 00 00 00 # <unused>
+FD E0FD . N FD FD FD 00 00 00 # <unused>
+FE E0FE . N FE FE FE 00 00 00 # <unused>
+FF E0FF . N FF FF FF 00 00 00 # <unused>
